src/test/java/com/gznznzjsn/outstagram/persistence/converter/LocalDateTimeTypeHandlerTest.java

```diff
@@ -0,0 +1,168 @@
+package com.gznznzjsn.outstagram.persistence.converter;
+
+import lombok.SneakyThrows;
+import org.apache.ibatis.type.JdbcType;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.CallableStatement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class LocalDateTimeTypeHandlerTest {
+
+    @InjectMocks
+    private LocalDateTimeTypeHandler handler;
+
+    @Test
+    @SneakyThrows
```

Comment:
@gznznzjsn in test you should just declare that this piece of code can throw an exception, don't catch anything